test/utils.js

```diff
@@ -7,8 +7,42 @@ import { Provider } from 'react-redux';
 import thunk from 'redux-thunk';
 import sinon from 'sinon';
 
+import _get from 'lodash/get';
+import toPath from 'lodash/toPath';
+import i from 'icepick';
+import Immutable from 'immutable';
+
 import { controls, modelReducer, formReducer, Control, actions } from '../src';
 
```

Comment:
Since I'm finding myself repeating the stuff below in each spec file I've been refactoring for the immutable strategy I thought maybe it would be helpful to put it here.  You can see in form-component-spec.js where I then used it like so:

```
const testContexts = {
  standard: {
    ...defaultTestContexts.standard,
    Form: _Form,
    modelReducer: _modelReducer,
    formReducer: _formReducer,
    Field: _Field,
    Control: _Control,
    actions: _actions,
  },
  immutable: {
    ...defaultTestContexts.immutable,
    Form: ImmutableForm,
    modelReducer: immutableModelReducer,
    formReducer: immutableFormReducer,
    Field: ImmutableField,
    Control: ImmutableControl,
    actions: immutableActions,
  },
};
```

Let me know if you don't like that idea though!